app/models/link.rb

```diff
@@ -0,0 +1,27 @@
+class Link < ActiveRecord::Base
+  belongs_to :link_set
+
+  include Replaceable
+  include DefaultAttributes
+
+  validate :link_is_valid
+
+private
+  def link_is_valid
+    # Test that:
+    # - the `link_type` attribute is valid
+    # - the `target_content_id` attribute is a UUID
```

Comment:
I'd say we do not need it, but the original version had a similar comment so I thought I should have left it there:
https://github.com/alphagov/publishing-api/blob/master/app/models/link_set.rb#L18

I am happy to remove it.